Assets/Scripts/Weapon.cs

```diff
@@ -11,6 +11,19 @@ public class Weapon : MonoBehaviour
 
         void ShootArrow()
         {
+            // TODO: Use real-time setting when inventory is ready
+            //
+            //PlayerController player = gameObject.GetComponent<PlayerController>();
+            //OperationType operation = player.GetOperationSelector().QueryCurrentItem().Operator;
```

Comment:
I have a PR out right now that selects the right thing, so you could use that if it gets merged first.

Reply:
No problem. I'll hook it up if it's available.